app/models/manageiq/providers/azure/cloud_manager/provision/cloning.rb

```diff
@@ -82,13 +80,22 @@ def prepare_for_clone_task
             :caching      => 'ReadWrite',
             :osType       => os
           }
-        },
-        :networkProfile  => {
-          :networkInterfaces => [{:id => nic_id}],
         }
       }
     }
 
+    # The -1 value is set in ProvisionWorkflow to distinguish between the
+    # desire for a new Public IP address vs a private IP.
```

Comment:
Is the word "private" correct here?

Reply:
Not sure what you're asking. If a Public IP isn't assigned, Azure will automatically assign a private IP.

Reply:
Ok, thanks

Reply:
In reality, just poking around on the Azure portal, it appears that NIC's with a public IP still have a private IP as well.